libs/community/langchain_community/llms/__init__.py

```diff
@@ -950,4 +959,5 @@ def get_type_to_cls_dict() -> Dict[str, Callable[[], Type[BaseLLM]]]:
         "yandex_gpt": _import_yandex_gpt,
         "yuan2": _import_yuan2,
         "VolcEngineMaasLLM": _import_volcengine_maas,
+        "SparkLLM": _import_sparkllm(),
```

Comment:
The `_import_sparkllm()` function should not be called here. It's breaking everything with the message:

"Did not find spark_app_id, please add an environment variable `IFLYTEK_SPARK_APP_ID` which contains it, or pass `spark_app_id` as a named parameter. (type=value_error)"

Reply:
> The `_import_sparkllm()` function should not be called here. It's breaking everything with the message:
> 
> "Did not find spark_app_id, please add an environment variable `IFLYTEK_SPARK_APP_ID` which contains it, or pass `spark_app_id` as a named parameter. (type=value_error)"

Thanks @drazvan , I will fix it.